images/miq-app/Dockerfile

```diff
@@ -1,19 +1,17 @@
-FROM centos:7
+FROM carbonin/manageiq-ruby-container
```

Comment:
The upstream build doesn't use SCL ruby at all, so that's a separate discussion... If there are other ruby containers we can use and we don't need to build our own, that'll be great!